File: server/models/nation_sb.js
const mongoose = require("mongoose");
const NationSchemaSb = new mongoose.Schema({
  date: Date,
  sb_battles_mean: Number,
  sb_air_frags_per_battle: Number,
  nation: String,
  sb_battles_sum: Number,
  sb_win_rate: Number,
  sb_air_frags_per_death: Number,
  sb_ground_frags_per_battle: Number,
  sb_ground_frags_per_death: Number,
  sb_lower_br: Number,
  cls: String,
});

const sb_ranks_all = mongoose.model("sb_ranks_all", NationSchemaSb, "sb_ranks_all");
const sb_ranks_1 = mongoose.model("sb_ranks_1", NationSchemaSb, "sb_ranks_1");
const sb_ranks_0 = mongoose.model("sb_ranks_0", NationSchemaSb, "sb_ranks_0");
module.exports = { sb_ranks_all, sb_ranks_1, sb_ranks_0 };

File: server/src/main/java/com/example/server/nationsb/NationSb.java
package com.example.server.nationsb;

import org.springframework.data.mongodb.core.mapping.Field;

import com.example.server.Nation;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public abstract class NationSb extends Nation {

    @Field("sb_lower_br")
    private Float lowerBr;
    @Field("sb_win_rate")
    private Float winRate;
    @Field("sb_battles_sum")
    private Float battleSum;
    @Field("sb_battles_mean")
    private Float battleMean;

    @Field("sb_ground_frags_per_battle")
    private Float groundFragPerBattle;
    @Field("sb_ground_frags_per_death")
    private Float groundFragPerDeath;

    @Field("sb_air_frags_per_battle")
    private Float airFragPerBattle;
    @Field("sb_air_frags_per_death")
    private Float airFragPerDeath;
}


